diff --git a/configure b/configure
index 6556df6..8b523c5 100755
--- a/configure
+++ b/configure
@@ -842,8 +842,8 @@ case "${ARCH}" in
             CRC_FOLDING_lo=""
         fi
 
-        CFLAGS="${CFLAGS} -DUSE_QUICK -DUSE_MEDIUM"
-        SFLAGS="${SFLAGS} -DUSE_QUICK -DUSE_MEDIUM"
+        CFLAGS="${CFLAGS} -DUSE_MEDIUM"
+        SFLAGS="${SFLAGS} -DUSE_MEDIUM"
 
         DEFLATE_QUICK_o="deflate_quick.o"
         DEFLATE_QUICK_lo="deflate_quick.lo"
@@ -888,8 +888,8 @@ case "${ARCH}" in
             CRC_FOLDING_lo=""
         fi
 
-        CFLAGS="${CFLAGS} -DUSE_QUICK -DUSE_MEDIUM"
-        SFLAGS="${SFLAGS} -DUSE_QUICK -DUSE_MEDIUM"
+        CFLAGS="${CFLAGS} -DUSE_MEDIUM"
+        SFLAGS="${SFLAGS} -DUSE_MEDIUM"
 
         DEFLATE_QUICK_o="deflate_quick.o"
         DEFLATE_QUICK_lo="deflate_quick.lo"
